### ISSUE TITLE
ahot will not build due to line truncation on gcc 5.2.0

### ISSUE THREAD

**Issue by crbates** [pyne/pyne#760]
I get the following error when trying to build pyne on gcc 5.2.0

``` sh
/pyne/src/transport_spatial_methods/3d/ahotn_kernel_module.f90:283:132:

    a22 = 1.0d0 + 2.0d0*ez*(dzx+dzy) + 2.0d0*gamma1*( (ex*ez-36.0d0*alpha1)*dxz/ex + (ey*ez-36.0d0*beta1 )*dyz/ey ) + 3.0d0*gamma0*ez0 
                                                                                                                                    1
Error: Line truncated at (1) [-Werror=line-truncation]
/pyne/src/transport_spatial_methods/3d/ahotn_kernel_module.f90:284:132:

    a33 = 1.0d0 + 2.0d0*ey*(dyx+dyz) + 2.0d0*beta1 *( (ex*ey-36.0d0*alpha1)*dxy/ex + (ey*ez-36.0d0*gamma1)*dzy/ez ) + 3.0d0*beta0 *ey0 
                                                                                                                                    1
Error: Line truncated at (1) [-Werror=line-truncation]
/pyne/src/transport_spatial_methods/3d/ahotn_kernel_module.f90:285:132:

    a44 = 1.0d0 + 2.0d0*ex*(dxz+dxy) + 2.0d0*alpha1*( (ex*ey-36.0d0*beta1 )*dyx/ey + (ex*ez-36.0d0*gamma1)*dzx/ez ) + 3.0d0*alpha0*ex0
```

**Comment by crbates** [comment]
This just requires 3 line continuations to be added to fix.

**Comment by scopatz** [comment]
Weird.  Thanks for reporting @crbates!

**Comment by rachelslaybaugh** [comment]
I think a PR that should be coming soon from @howland includes this fix...

Correction; this has already been resolved through PR #768 / #767
